### PR TITLE
docs: about

### PR DESCRIPTION
**Pull request by Shinigami92**
This PR is based on #658, so please merge that one first (done :heavy_check_mark:)

### CODE REVIEW

**Comment by ST-DDT** [comment]
This PRs title is misleading

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -14,15 +27,26 @@ const nav = [
...
+  '/about': [
     {
-      text: 'Guide',
+      text: 'About',
       children: [
         {
-          text: 'Recent Statement and FAQs',
-          link: '/update.html',
+          text: 'Announcements',
+          link: '/about/announcements',
```
Comment (ST-DDT) [inline docs/.vitepress/config.ts:37]: Shouldn't be `Announcements` its own top level item?

File: docs/about/team/members.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    "gitlab": "griest",
+    "role": ["Code Reviewer"]
+  },
+  {
+    "name": "Mateus Dadalto",
+    "github": "MateusDadalto",
+    "twitter": "MateusD",
+    "role": []
+  }
+]
```
Comment (ST-DDT) [inline docs/about/team/members.json:55]: IMO we should omit the resigned ones and add the new ones.

Comment (ST-DDT) [reply docs/about/team/members.json:55]: Maybe honor them in a "previous members" list.

**Review by ST-DDT — CHANGES_REQUESTED** [review]
Don't forget the redirect for update

**Comment by ST-DDT** [comment]
TODO for later: Consider adding SSR for team/api-docs to serve only static pages.

File: docs/about/team.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: Team
+---
+
+<script setup>
+  import TeamPage from './team/TeamPage.vue'
+</script>
+
+<TeamPage />
```
[the remark this answers is not in the document]
Comment (Shinigami92) [reply docs/about/team.md:9]: I'm not sure, but sadly I assume no
I orientate myself on https://github.com/vuejs/docs/tree/main/src/about

File: docs/.vitepress/config.ts (hunk 1 above)
Comment (Shinigami92) [reply docs/.vitepress/config.ts:37]: I'm not sure about that yet, I don't want to explode the nav at the top in the docs

File: docs/about/team/members.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
+[
```
[the remark this answers is not in the document]
Comment (Shinigami92) [reply docs/about/team/members.json:1]: I orientate myself on [https://github.com/vuejs/docs/tree/main/src/about](https://github.com/vuejs/docs/tree/main/src/about)

**Comment by Shinigami92** [comment]
> TODO for later: Consider adding SSR for team/api-docs to serve only static pages.

We found out that VitePress already does successfully SSG for the about section, but the API section seems not to work correctly 🤔 
But this is not part for this PR

File: docs/about/team/members.json [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+      "twitter": "clarkerican",
+      "roles": []
+    },
+    {
+      "name": "griest",
+      "github": "griest024",
+      "gitlab": "griest",
+      "roles": ["Code Reviewer"]
+    }
+  ],
```
Comment (Shinigami92) [inline docs/about/team/members.json:44]: Do we want any kind of order for core?
What should be a sort criteria?

File: docs/about/team/members.json [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+      "name": "Christopher Quadflieg",
+      "github": "Shinigami92",
+      "twitter": "Shini_92",
+      "roles": ["Code Maintainer"]
+    },
+    {
+      "name": "Daniel Bannert",
+      "github": "prisis",
+      "twitter": "_prisis_",
+      "roles": []
```
Comment (Shinigami92) [inline docs/about/team/members.json:30]: @prisis What are your roles? 🙂

File: docs/about/team/members.json [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+    },
+    {
+      "name": "Erica Clark",
+      "github": "clarkerican",
+      "twitter": "clarkerican",
+      "roles": []
```
Comment (Shinigami92) [inline docs/about/team/members.json:36]: @ericjeker What are your roles? 🙂

Comment (ericjeker) [reply docs/about/team/members.json:36]: That's probably not for me. 😂

Comment (Shinigami92) [reply docs/about/team/members.json:36]: Ups... sorry, just hit enter to early 😅

@clarkerican ... 👋

**Comment by Shinigami92** [comment]
Currently it looks like this (4k screen):

image


Except for the roles and maybe some minor improvements here and there, I would like to call this as a good start for the team page.
You are welcome to improve it in additional PRs in the future 💫

File: docs/about/team/TeamMember.vue [hunk 8]
```diff
@@ -0,0 +1,72 @@
+<script lang="ts" setup>
+import type { Member } from './TeamMember';
+
+defineProps<{ member: Member }>();
+</script>
+
+<template>
+  <div class="TeamMember">
+    <div class="avatar">
+      <img :src="'https://github.com/' + member.github + '.png'" width="140" />
```
Comment (import-brain) [inline docs/about/team/TeamMember.vue:10]: These pictures look massive on my screen, perhaps change the width to 110 or 120. Just personal preference though

Comment (import-brain) [reply docs/about/team/TeamMember.vue:10]: Other than this, this LGTM! Thank you so much for putting us on this page :)

File: docs/about/team/members.json [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+    {
+      "name": "Jessica Sachs",
+      "github": "JessicaSachs",
+      "twitter": "_JessicaSachs",
+      "roles": ["Press Officer"]
+    },
+    {
+      "name": "Daniel Theuke",
+      "github": "ST-DDT",
+      "roles": ["Code Maintainer", "Docs Automation"]
```
Comment (ST-DDT) [inline docs/about/team/members.json:18]:
```suggestion
      "roles": ["Code Maintainer", "Docs Automation"]
```

We should probably use no breaking spaces here (and in all role names):

<details>
<summary><strong>Old</strong></summary>

grafik

</details>

<details>
<summary><strong>New</strong></summary>

grafik

</details>

File: netlify.toml [hunk 10]
```diff
@@ -20,3 +20,10 @@
   to = "https://stackblitz.com/edit/faker-js-demo?file=index.ts"
   status = 301
   force = true
+
+# Redirect old update page to new announcement section
+[[redirects]]
+  from = "/update.html"
+  to = "https://chat.fakerjs.dev/about/2022-01-14.html"
```
Comment (ST-DDT) [inline netlify.toml:27]:
```suggestion
  to = "https://fakerjs.dev/about/2022-01-14.html"
```

?

File: docs/.vitepress/config.ts [hunk 11]
```diff
@@ -4,26 +4,50 @@ import { apiPages } from './api-pages';
...
+    items: [
+      {
+        text: 'Announcements',
+        link: '/about/announcements',
+      },
+      {
+        text: 'Team',
+        link: '/about/team',
+      },
     ],
```
Comment (ST-DDT) [inline docs/.vitepress/config.ts:27]: I don't like that I cannot click on just `About` and some page (e.g. Announcements) opens.
Also the `About` sidenav looks very empty.
IMO we should at least list all announcements when `Announcements` is expanded (Similar to the api docs pages).

File: docs/about/announcements.md [hunk 12]
```diff
@@ -0,0 +1,7 @@
+---
+editLink: false
+---
+
+# Announcements
+
+- [2022-01-14 - An update from the Faker team](./2022-01-14)
```
Comment (ST-DDT) [inline docs/about/announcements.md:7]: IMO we should throw the announcements in their own folder (to ease a future automatic index/sidenav generation)

File: docs/about/team/TeamMember.vue (hunk 8 above)
Comment (ST-DDT) [reply docs/about/team/TeamMember.vue:10]: I agree with @import-brain

File: netlify.toml (hunk 10 above)
Comment (Shinigami92) [reply netlify.toml:27]: I totally not copied the above block, how dare I :eyes:

**Review by import-brain — APPROVED** [review]
LGTM, thank you! :)

**Review by import-brain — APPROVED** [review]

**Review by pkuczynski — DISMISSED** [review]

**Review by ST-DDT — APPROVED** [review]
IMO clicking on about should already redirect the user, unfortunately vitepress doesn't support this or I don't know how.
Also the announcement page is quite empty. Maybe we can add some more pseudo content here. Or even a v6 release anouncement.

**Comment by Shinigami92** [comment]
> IMO clicking on about should already redirect the user, unfortunately vitepress doesn't support this or I don't know how.
Also the announcement page is quite empty. Maybe we can add some more pseudo content here. Or even a v6 release announcement.

Yeah, I hope that this page will be filled in the future with some more content. Right now it's just the start and that is one reason why it's so empty.

I would like to leave this to @JessicaSachs

**Review by import-brain — APPROVED** [review]

**Review by prisis — APPROVED** [review]

File: docs/about/team/members.json (hunk 5 above)
Comment (prisis) [reply docs/about/team/members.json]: alphabetically, easy and we have a order without personal issues

**Review by import-brain — APPROVED** [review]

**Review by ST-DDT — APPROVED** [review]

File: docs/about/team/members.json (hunk 7 above)
Comment (Shinigami92) [reply docs/about/team/members.json:30]: Will merge now, please edit on demand in a separate PR

**Pull request merged**